src/kontrol/kdist/cheatcodes.md

```diff
@@ -1691,6 +1711,8 @@ Selectors for **implemented** cheat code functions.
     rule ( selector ( "mockCall(address,bytes,bytes)" )            => 3110212580 )
     rule ( selector ( "mockFunction(address,address,bytes)" )      => 2918731041 )
     rule ( selector ( "copyStorage(address,address)" )             => 540912653  )
+    rule ( selector ( "replaceConstraint(uint256,uint8,uint256,bool)" ) => 1408652492 )
```

Comment:
We should probably remove it or at least move to the "unimplemented cheatcode" section:
```suggestion
```